airbyte-integrations/connectors/source-kafka/build.gradle

```diff
@@ -9,13 +9,24 @@ application {
     applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
 }
 
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://packages.confluent.io/maven"
```

Comment:
@sivankumar86 You need to update gradle dependencies as per `docs/contributing-to-airbyte/gradle-dependency-update.md` in the file `deps.toml` in the project root. An example is the package `com.amazonaws:aws-java-sdk-s3` which is in the toml and used by the `destination-s3` connector

Reply:
It seems, it is only for common library and above is specific to kafka. let me know if you still want to move to root because, it does not make sense.